documentation/config_file_ca.md

```diff
@@ -8,33 +8,31 @@ canonical: "/puppetserver/latest/config_file_ca.html"
+The `certificate-status` setting in `ca.conf` provides [deprecated][] configuration options for access to the `certificate_status` and `certificate_statuses` HTTP endpoints. These endpoints allow certificates to be signed, revoked, and deleted through HTTP requests, which provides full control over Puppet's ability to securely authorize access. Therefore, you should **always** restrict access to `ca.conf`.
 
-> **Puppet Enterprise Note:** Puppet Enterprise uses these endpoints to provide a console interface for certificate signing. For more information, see the [Certificate Status documentation](/puppet/latest/reference/http_api/http_certificate_status.html).
+> **Puppet Enterprise Note:** Puppet Enterprise uses these endpoints to provide a console interface for certificate signing. For more information, see [Certificate Status ](/puppet/latest/reference/http_api/http_certificate_status.html).
 
-This setting takes two parameters: `authorization-required` and `client-whitelist`. If `authorization-required` is not set or set to `true` and `client-whitelist` is not set or set to an empty list, Puppet Server uses the [new authorization methods][`trapperkeeper-authorization`] and [new `auth.conf`][] format introduced in Puppet Server 2.2 to control access to the Puppet Server administration API endpoints.
+The `certificate-status` setting takes two parameters: `authorization-required` and `client-whitelist`. If `authorization-required` is set to `true` or not set, **and** `client-whitelist` is set to an empty list or not set, Puppet Server uses the [authorization methods][`trapperkeeper-authorization`] and [`auth.conf`][] format introduced in Puppet Server 2.2 to control access to the administration API endpoints.
 
 * `authorization-required` determines whether a client certificate is required to access certificate status endpoints. If this parameter is set to `false`, all requests can access this API. If set to `true`, only the clients whose certificate names are included in the `client-whitelist` setting can access the admin API. If this parameter is not specified but the `client-whitelist` parameter is, this parameter's value defaults to `true`.
 * `client-whitelist` contains a list of client certificate names that are whitelisted for access to the certificate status endpoints. Puppet Server denies access to requests at these endpoints that do not present a valid client certificate named in this list.
 
-## Example (Legacy)
+## Example (Deprecated)
 
-If you are not using the new authorization methods, follow this structure to configure  `certificate_status` and `certificate_statuses` endpoint access in `ca.conf`:
+If you are using the deprecated authorization methods, follow this structure to configure  `certificate_status` and `certificate_statuses` endpoint access in `ca.conf`, whitelisting a client named `host`:
```

Comment:
Should either change the reference in the text here to `host1` or in the HOCON example to `host`.